Error: `for` must be an `id` reference to a form control and may not reference other element types. To create the intended relationship, the `div` must have a `role` of "group" and an `aria-labelledby` attribute that references the `label`. Or use `fieldset` and `legend`.
Error: Use `aria-describedby` to reference a group description.
Error: For usability, it is recommended to put the group description at the top of the component, just after the label.
Error: Checkboxes in a group should use a `name` value that ends with "[]" to allow server-side code to collect the values as an array (or similar).
Error: `aria-invalid="true"` is used to accessibly indicate fields that have errors.
Error: When a checkbox is marked as `required`, it must be checked to be valid. Per the instructions, the user only needs to choose one or more, so `required` must not be used on any of the checkbox inputs.
Error: Color alone should not be used to convey errors.
Error: The error text should not be hidden when the component has an error.
diff --git a/output/Checkbox Group with Validation Error/4/488c558e-b7ec-49a6-8b26-80173d73ea13.html b/output/Checkbox Group with Validation Error/4/488c558e-b7ec-49a6-8b26-80173d73ea13.html
index c962315..84a3aaa 100644
--- a/output/Checkbox Group with Validation Error/4/488c558e-b7ec-49a6-8b26-80173d73ea13.html	
+++ b/output/Checkbox Group with Validation Error/4/488c558e-b7ec-49a6-8b26-80173d73ea13.html	
@@ -1,16 +1,16 @@
-<label for="fruits">What fruits do you like?</label>
-<div id="fruits" role="group" aria-labelledby="fruits">
+<label id="fruits">What fruits do you like?</label>
+<span id="fruits-error">You must choose one or more fruits</span>
+<div role="group" aria-labelledby="fruits" aria-describedby="fruits-error">
   <div>
-    <input type="checkbox" id="bananas" name="fruits" value="bananas" required>
+    <input type="checkbox" id="bananas" name="fruits[]" value="bananas" aria-invalid="true">
     <label for="bananas">Bananas</label>
   </div>
   <div>
-    <input type="checkbox" id="oranges" name="fruits" value="oranges" required>
+    <input type="checkbox" id="oranges" name="fruits[]" value="oranges" aria-invalid="true">
     <label for="oranges">Oranges</label>
   </div>
   <div>
-    <input type="checkbox" id="apples" name="fruits" value="apples" required>
+    <input type="checkbox" id="apples" name="fruits[]" value="apples" aria-invalid="true">
     <label for="apples">Apples</label>
   </div>
-</div>
-<span id="fruits-error" style="color: red; display: none;">You must choose one or more fruits</span>
\ No newline at end of file
+</div>
\ No newline at end of file
